packages/flutter/lib/src/material/tabs.dart

```diff
@@ -331,6 +331,7 @@ class _IndicatorPainter extends CustomPainter {
     required _IndicatorPainter? old,
     required this.indicatorPadding,
     this.dividerColor,
+    required this.labelPaddings,
```

Comment:
Can you put this before the `dividerColor` to place all required parameters first?

Reply:
done